pkg/analyzer/tui/form.go

```diff
@@ -0,0 +1,89 @@
+package tui
+
+import (
+	"fmt"
+	"slices"
+
+	"github.com/charmbracelet/bubbles/key"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/analyzer/analyzers"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/tui/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/tui/components/textinputs"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/tui/styles"
+)
+
+type FormPage struct {
+	Common  *common.Common
+	KeyType string
+	form    textinputs.Model
+}
+
+func NewFormPage(c *common.Common, keyType string) FormPage {
+	inputs := []textinputs.InputConfig{{
+		Label:       "Secret",
+		Key:         "key",
+		Required:    true,
+		RedactInput: true,
+	}}
+	if keyType == "shopify" {
+		inputs = append(inputs, textinputs.InputConfig{
+			Label:    "Shopify URL",
+			Key:      "url",
+			Required: true,
+		})
+	}
+
+	form := textinputs.New(inputs).
+		SetHeader(titleStyle.Render(fmt.Sprintf("Configuring %s analyzer", keyType))).
+		SetFooter("⚠️  Running TruffleHog Analyze will send a lot of requests ⚠️").
```

Comment:
Can we make this dependent on an attribute on the analyzers? Not all do this.

Reply:
Is there an existing attribute I can reference? If not, do you have a list of analyzers that do brute-force?

Also open to phrasing suggestions. We could change it to something like "Running TruffleHog Analyze may send a lot of requests"

Reply:
We'd have to look at the analyzers and define that attribute.